zio/detector/ztests/tzng-nonewline.yaml

```diff
@@ -0,0 +1,9 @@
+zql: '*'
+
+input: |-
```

Comment:
This was actually a material change I made to a test while I was in here. The way things were when I started this exercise, `zng.yaml` and `zng-nonewline.yaml` were byte-for-byte exactly the same, so the "nonewline" variant was not testing what was intended. I added the `-` here, following the pattern of the other "nonewline" tests.